src/content/about/governance.mdx

```diff
@@ -223,7 +223,7 @@ nf-core maintainers will have write access to repositories.
 <Profile username="ggabernet" affiliation="Quantitative Biology Center, QBiC, University of Tübingen">
```

Comment:
@ggabernet also has a new affiliation

Reply:
Thanks! I'll merge for now and leave Gisela to make her updates- maybe we should ask everyone to double check theirs